sycl/include/sycl/detail/group_sort_impl.hpp

```diff
@@ -71,19 +71,14 @@ struct GetValueType<sycl::multi_ptr<ElementType, Space, IsDecorated>> {
 // since we couldn't assign data to raw memory, it's better to use placement
 // for first assignment
 template <typename Acc, typename T>
-void set_value(Acc ptr, const size_t idx, const T &val, bool is_first) {
-  if (is_first) {
-    ::new (ptr + idx) T(val);
-  } else {
-    ptr[idx] = val;
-  }
+void set_value(Acc ptr, const size_t idx, const T &val) {
```

Comment:
I don't think we need this helper anymore. Sorry for not noticing in the previous review round.

Reply:
I agree, removed.